downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/CreateReactome2BioSystems.java

```diff
@@ -0,0 +1,28 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.biosystems.ReactomeToBioSystemsConverter;
+
+public class CreateReactome2BioSystems {
+	private static final Logger logger = LogManager.getLogger();
+	public static void execute(String host, String database, String username, String password, int port, String releaseNumber) throws IOException {
+		
+		logger.info("Running CreateReactome2BioSystems step");
+		// The last argument, 'BioSystems', specifies the output directory of the ReactomeToBioSystems.zip file. 
+		// The script removes all files within the named directory, so only change the output directory to an empty or non-existent one
+		String bioSystemsDirectory = "BioSystems";
```

Comment:
Maybe declare this at the start of the execute method as a constant:

`final String BIO_SYSTEMS_DIRECTORY = "BioSystems";`

Reply:
Done